src/commands/StatsCommand.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  SlashCommandBuilder,
+  ChatInputCommandInteraction,
+  EmbedBuilder,
+  SlashCommandSubcommandsOnlyBuilder,
+} from "discord.js";
+import { Command } from "./CommandInterface";
+import { PlayerData } from "../database/PlayerData";
+import { EloUtil } from "../util/EloUtil";
+import { log } from "console";
+
+export default class StatsCommand implements Command {
+  name: string;
+  description: string;
+  data: SlashCommandBuilder;
+
+  constructor() {
+    this.name = "stats";
+    this.description = "Get the stats of yourself or another player.";
+
+    this.data = new SlashCommandBuilder()
+      .setName(this.name)
+      .setDescription(this.description)
+      .addStringOption((option) =>
+        option
+          .setName("player")
```

Comment:
We are only searching for player, what if a player changes name and someone else takes their old name for example, we need to be able to use discord ID. I'm thinking it should actually be the primary identifier and player is secondary